openapi.yaml

```diff
@@ -22,21 +22,22 @@ consumes:
 produces:
   - application/json
 paths:
-  /onbase/{osuId}:
+  /onbase/admissions/{osuId}:
     get:
-      summary: Return a recordset of application data for an OSU ID
+      summary: Return an admission data for an OSU ID
       tags:
         - onBase
-      description: Return a recordset of application data for an OSU ID
-      operationId: getApplications
+        - admissions
```

Comment:
@Ruefa I usually tag multiple tags if the endpoint contains multiple stuff. For example: https://github.com/osu-mist/students-api/blob/2ad5a02bb99043bc7ae8ca2571fd836ea2a8a300/openapi.yaml#L25-L27

Here, it means the current endpoint (`admissions`) is one of the resources under `onBase` API. We'll probably want to tag more than one tags to the same endpoint in the future if we start using relationships, e.g. `/pizzas` endpoint can have `pizzas` and all other related resources as tags.

Reply:
That makes sense when looking at the openapi code but it seems confusing to me when looking at the generated documentation. 
![image](https://user-images.githubusercontent.com/16888647/68166721-177ee180-ff18-11e9-8eda-a615d893dc5b.png)

Reply:
Also, since we'll add a new resource to this API, the openapi will eventually look like this:

![Screen Shot 2019-11-04 at 3 31 13 PM](https://user-images.githubusercontent.com/9169491/68166788-4c8b3400-ff18-11e9-83a2-59b906674c43.png)

Reply:
Ah that looks better.